webapp/src/ts/services/form2sms.service.ts

```diff
@@ -0,0 +1,61 @@
+import * as odkForm2sms from 'odk-xform-compact-record-representation-for-sms'
+import { Injectable } from '@angular/core';
+
+import { DbService } from '@mm-services/db.service';
+import { GetReportContentService } from '@mm-services/get-report-content.service';
+import { ParseProvider } from '@mm-providers/parse.provider';
+
+
+const concat = (...args) => args.join('');
```

Comment:
Can these functions: concat, spaced and match be inside the service class or maybe they can be in a utils provider/service?

Reply:
I guess they can, I want to refactor as little as possible though. Do you think it's a necessary change?

Reply:
Refactored!